_lzbench/lzbench.h

```diff
@@ -212,15 +209,16 @@ static const compressor_desc_t comp_desc[LZBENCH_COMPRESSOR_COUNT] =
     { "zstd22LDM",  "1.4.9",       1,  22,   22,       0, lzbench_zstd_LDM_compress,   lzbench_zstd_decompress,       lzbench_zstd_LDM_init,   lzbench_zstd_deinit },
     { "zstd24LDM",  "1.4.9",       1,  22,   24,       0, lzbench_zstd_LDM_compress,   lzbench_zstd_decompress,       lzbench_zstd_LDM_init,   lzbench_zstd_deinit },
     { "nakamichi",  "okamigan",    0,   0,    0,       0, lzbench_nakamichi_compress,  lzbench_nakamichi_decompress,  NULL,                    NULL },
-    { "cudaMemcpy", "",            0,   0,    0,       0, lzbench_return_0,            lzbench_cuda_memcpy,           lzbench_cuda_init,       lzbench_cuda_deinit },
+#ifdef BENCH_HAS_CUDA
+    { "cudaMemcpy", "",            0,   0,    0,       0, lzbench_cuda_return_0,       lzbench_cuda_memcpy,           lzbench_cuda_init,       lzbench_cuda_deinit },
     { "nvcomp_lz4", "1.2.2",       0,   5,    0,       0, lzbench_nvcomp_compress,     lzbench_nvcomp_decompress,     lzbench_nvcomp_init,     lzbench_nvcomp_deinit },
+#endif
```

Comment:
Either way, I don't see any problems in removing the `#ifdef BENCH_HAS_CUDA` here.
The rest of the discussion is certainly more general than this PR.